### PR TITLE
Update demographic form and contact page

### PR DESCRIPTION
**Pull request by kimberscott**
Refs: https://openscience.atlassian.net/browse/LEI-

## Purpose
Finalize demographic survey questions and give researchers a way to learn more about status of project

## Summary of changes
* Added three questions to demographic form - country, state if in US, and urban/suburban/rural
* Use updated rounding procedure for children's ages based on birthdates in demographic survey, as in child registration
* Update contact text for researchers

## Testing notes

* Tested in Firefox and Chrome
* State question appears/disappears appropriately based on whether country is USA
* New fields in demographic form are saved - shown again when user returns to survey, and accessible via client.py from jamdb collection, including after edits

### CODE REVIEW

File: app/templates/my/demographics.hbs [hunk 1]
```diff
@@ -39,7 +80,7 @@
                 {{pikaday-input value=(unbound bd) onPikadaySelect=(action 'setChildBirthday' index) format="MM/DD/YYYY" yearRange="1990,currentYear" maxDate=today class="form-control"}}
                 <span class="input-group-addon">
```
Comment (abought) [inline app/templates/my/demographics.hbs]: Note from functional review: an empty grey box appears here if there is no birthday. I'll address in a followup PR as that is not related to this change.

File: app/templates/my/demographics.hbs [hunk 2]
```diff
@@ -1,10 +1,51 @@
...
+          {{#each countryOptions as |countryOption|}}
+            <option value={{countryOption.code}} selected={{eq model.demographicsCountry countryOption.code}}>{{countryOption.country}}</option>
+          {{/each}}
+      </select>
+      {{/bs-form-group}}
+
+
+      {{#if (eq model.demographicsCountry "US") }}
+      {{#bs-form-group}}
+      <label>What state do you live in?</label>
```
Comment (abought) [inline app/templates/my/demographics.hbs]: Small edge case note:

- If the user selects a US state, but later moves to another country, then the US State selected will not be cleared. (Thus it is possible to be listed as a resident of Alaska, Afghanistan)

We'll leave this as is for now, but just a heads up. That's probably a very rare event.

File: app/templates/my/demographics.hbs [hunk 3]
```diff
@@ -1,10 +1,51 @@
...
+        <option value="" disabled selected hidden>Please choose...</option>
+          {{#each stateOptions as |stateOption|}}
+            <option value={{stateOption.code}} selected={{eq model.demographicsState stateOption.code}}>{{stateOption.state}}</option>
+          {{/each}}
+      </select>
+      {{/bs-form-group}}
+      {{/if}}
+
+      {{#bs-form-group}}
+      <label>How would you describe the area where you live?</label>
```
Comment (abought) [inline app/templates/my/demographics.hbs]: Tiny future improvement: we generally try to indent nested sections in markup for readability. I'll tweak that while I make a few polish changes to the demographics page later today. (the changes are unrelated to this code but need to be made)

File: app/templates/contact.hbs [hunk 4]
```diff
@@ -14,9 +14,11 @@
...
+
+        <p>We're not quite ready to start hosting additional studies from other labs yet, but our partners at the Center for Open Science are actively developing functionality for multiple labs to create and post their own studies on Lookit. Although we aim for the platform to be as accessible as possible, some familiarity with programming basics will be helpful for researchers. An ideal skillset for creating your own studies would include basic understanding of HTML, CSS, and JavaScript for study creation; passing familiarity with the <a href="https://guides.emberjs.com/v2.9.0/" target=_blank>Ember framework</a> used by Lookit and Experimenter; and python for data processing.</p>
```
Comment (kimberscott) [inline app/templates/contact.hbs]: Thanks!

File: app/templates/my/demographics.hbs (hunk 2 above)
Comment (kimberscott) [inline app/templates/my/demographics.hbs]: Thanks! Oops, I thought I'd made a note of this--we're just expecting that the state field is only meaningful if the country is US.

**Comment by abought** [comment]
Just to close the loop, I made the followup changes in #73 . I think that covers all followup notes, but let me know if further is needed.

**Pull request merged**